setup.py

```diff
@@ -513,6 +514,17 @@ def configuration(parent_package='', top_path=None):
 
 
 def setup_package():
+    np_minversion = '1.16.5'
+    np_maxversion = '1.23.0'
+    python_minversion = '3.7'
+    python_maxversion = '3.9'
```

Comment:
Sorry, I told you the wrong thing. It's `<` not `<=` so it should be 3.10. This failed one Azure job. 

The `maxversion` name is slightly suboptimal, but it's hard to come up with a better one `upperbound_openinterval` is a bit much